src/js/services/involvement-API/getLikes.js

```diff
@@ -0,0 +1,11 @@
+import involvementAPI from './involvement-API.js';
+
+// Function to get the likes of each Show  -----------------------------------------------------
+// eslint-disable-next-line consistent-return
```

Comment:
- Please, let's remove this eslint disable as it has no effect on your code also, it's not a good practice disabling the linter as this helps to improve our development skills.

Reply:
@zieeco, 🙏🏼 thank you to point this out! We have overlooked it, again, we will the changes required right away! Cheers, and happy coding!